tracing/src/main/java/org/axonframework/extensions/tracing/TracingQueryGateway.java

```diff
@@ -84,13 +85,13 @@ public static Builder builder() {
 
     @Override
     public <R, Q> CompletableFuture<R> query(String queryName, Q query, ResponseType<R> responseType) {
-        Span parentSpan = tracer.activeSpan();
-        Span span = tracer.buildSpan("send_" + SpanUtils.messageName(query.getClass(), queryName)).start();
-        try (Scope ignored = tracer.activateSpan(span)) {
-            return delegate.query(queryName, query, responseType).whenComplete((r, e) -> {
-                span.finish();
-            });
-        }
+        QueryMessage qry = new GenericQueryMessage(query, queryName, responseType);
+        return getWithSpan("send_" + SpanUtils.messageName(qry), (childSpan) ->
```

Comment:
I know it was called `send_` prior to this change, but it might be more in line with the Open Tracing Standard to rename this to `query_`

Reply:
Indeed it makes more sense and I will udpate it.